pynitrokey/fido2/client.py

```diff
@@ -347,6 +347,10 @@ def simple_secret(
 
         return output
 
+    def has_pin_set(self) -> bool:
+        assert isinstance(self.client, Fido2Client)
```

Comment:
I think `assert self.client is not None` would be easier to understand.

Reply:
affirmitive! done